### DISCUSSION TITLE
Secure setup of Deno's "Persist data using Firebase" example

### DISCUSSION THREAD

**Discussion by milotheirself** [firebase/firebase-admin-node#1820] [Q&A]
[Deno's example of using Firebase](https://deno.com/deploy/docs/tutorial-firebase) is currently relying on the Firebase web libraries rather than the Firebase Admin SDK, which would require to set up an email-user as an admin with access to parts of the database to manage them.

<br>

And I basically have two questions now: 

1. Is this even safe to begin with?
2. What would the Firestore rules with such a user look like?

<br>

---

Here is a working version of the example: [https://gist.github.com/milotheirself/3ede9570f26a42babe7e40c58b8b0445](https://gist.github.com/milotheirself/3ede9570f26a42babe7e40c58b8b0445).

And these are the rules, that I assume are required for it:

```
service cloud.firestore {
  match /databases/{database}/documents {
    match /songs/{document=**} {
      allow read, write: if request.auth.uid == '****************************';
    }
  }
}
```

**Comment by akauppi** [comment]
I happened to read this, and skimmed through the link.

You might consider asking this kind of questions in the Firebase Developers Discord service (of course if anyone covers them here, fine!). It’s a community approach whereas this one is official by Firebase. Can provide you an invite if you want - and happy to help further, but the link raised some other questions as well… So it’ll be a long thread.

**Reply by milotheirself** [reply]
Yes, it would be great if you could send me an invite link. Thank you! ^^

**Reply by akauppi** [reply]
@milotheirself You can google it: https://discord.com/invite/BN2cgc3

Once there, give me a message (asko)
